### ISSUE TITLE
Alt text guidelines for images in docs

### ISSUE THREAD

**Issue by LauraLangdon** [elk-zone/elk#2138]
I'm looking at the READ.ME for the docs, and in the [images section](https://github.com/elk-zone/elk/tree/main/docs#images), and I noticed there isn't a guideline regarding the use of alt text. We could provide guidelines stipulating that alt text should be used, and a link for info on [how to write good alt text](https://accessibility.huit.harvard.edu/describe-content-images).
